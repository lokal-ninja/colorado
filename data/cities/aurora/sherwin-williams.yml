id: 4168517910
type: node
shop: doityourself
latitude: 39.661441700000005
longitude: -104.85255670000001
postcode: "80014"
city: Aurora
street: "South Parker Road"
housenumber: 3106
phone: ""
opening_hours: "Mo-Fr 07:00-19:00, Sa 08:00-18:00, Su 10:00-18:00"
website: ""
