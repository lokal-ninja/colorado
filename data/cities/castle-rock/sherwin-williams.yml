id: 222031155
type: node
latitude: 39.36568
longitude: -104.86109
postcode: "80104"
city: Castle Rock
street: Plum Creek Parkway
housenumber: "174"
phone: ""
opening_hours: ""
website: ""
