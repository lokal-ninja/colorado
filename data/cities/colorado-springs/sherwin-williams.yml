id: 8397841858
type: node
latitude: 38.8082
longitude: -104.82174
postcode: ""
city: Colorado Springs
street: "South Nevada Avenue"
housenumber: 1815
phone: ""
opening_hours: ""
website: ""