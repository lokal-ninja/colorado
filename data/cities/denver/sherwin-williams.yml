id: 2930372078
type: node
latitude: 39.678819100000005
longitude: -104.93234600000001
postcode: "80222"
city: Denver
street: "East Evans Avenue"
housenumber: 4697
phone: "303-756-9447"
opening_hours: ""
website: "https://www.sherwin-williams.com/"
