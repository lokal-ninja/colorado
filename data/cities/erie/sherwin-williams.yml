id: 831267663
type: way
latitude: 40.00084
longitude: -105.01488
postcode: "80516"
city: Erie
street: "CO-7"
housenumber: 2325
phone: ""
opening_hours: ""
website: ""