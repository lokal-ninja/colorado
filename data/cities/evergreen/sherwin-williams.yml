id: 203936472
type: way
shop: paint
latitude: 39.700369200000004
longitude: -105.3254077
postcode: "80439"
city: Evergreen
street: "Swede Gulch Road"
housenumber: 1002
phone: ""
opening_hours: "Mo-Fr 07:00-19:00; Sa 08:00-17:00; Su 10:00-16:00"
website: "https://sherwin-williams.com"
