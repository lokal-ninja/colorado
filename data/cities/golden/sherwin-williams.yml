id: 6944107701
type: node
latitude: 39.73765
longitude: -105.19223
postcode: "80401"
city: Golden
street: South Golden Road
housenumber: "17101"
phone: +1-303-279-9101
opening_hours: Mo-Fr 06:00-18:00; Sa 08:00-17:00; Su 10:00-16:00
website: https://www.sherwin-williams.com/store-locator/paint-store/golden/co/727692
